src/tree.rs

```diff
@@ -1419,13 +1430,31 @@ impl ProblemCounts {
     }
 }
 
+/// A record for an item or tombstone in a bookmark tree.
+#[derive(Clone, Copy, Debug)]
+pub enum Record<'t> {
```

Comment:
I can't think of a better name for this. `ItemOrTombstone` is too wordy, and `Entry` is already taken. @mhammond, any ideas? 😄

Reply:
I got rid of this in the latest refactor. Problem solved! 😆